### DISCUSSION TITLE
How to configure cypress/react/plugins/next to DISABLE react-dev-tools

### DISCUSSION THREAD

**Discussion by glomotion** [cypress-io/cypress#20285] [Component Testing]
So we're encountering some incredible lag whilst using the `cypress open-ct` test runner locally (even on top spec m1max macbooks). Often it can take the dev tools JS console upwards of 2 minutes to catch up with what is seen inside of the editor.

My theory is that this lag is largely due to react-dev-tools running within the page. I was hoping there might be a way to simply disable the react-dev-tools from running, but so far, i have had no luck doing so. :(

Our setup for the component tests is pretty stock standard: 

```javascript
// cypress/plugins/index.js

const injectDevServer = require('@cypress/react/plugins/next');
const { loadEnvConfig } = require('@next/env');

module.exports = (on, config) => {
  require('@cypress/code-coverage/task')(on, config);

  // loadEnvConfig tells Next to load the .env.test file as it defaults to productions 🤨
  const { combinedEnv } = loadEnvConfig(process.cwd(), true);
  config.env = Object.assign(config.env, combinedEnv);

  if (config.testingType === 'component') {
    injectDevServer(on, config);
  }

  return config; 
};

```

Has anyone been successful in disabling the react-dev-tools?

**Comment by cubanx** [comment]
I also would love an answer to this question. Currently, it's not `cypress` fault, but React Dev Tools is throwing this error CONSTANTLY:
`Cannot remove node 1738 because no matching node was found in the Store.`

Where the number changes.

It's a known issue with React Dev Tools:
https://github.com/facebook/react/issues/23202

There are many people reporting the problem.
